src/components/ec-datepicker/ec-datepicker.story.js

```diff
@@ -53,6 +77,8 @@ export const basic = (args, { argTypes }) => ({
 
       <button @click="toggleWeekendAvailability">Toggle weekend availability</button>
       <button @click="changeDisabledDates('2022-02-22')">Disable 2022-02-22 UTC</button>
+      <button @click="model = new Date('2022-02-25')">Select 2022-02-25 date</button>
```

Comment:
The flatpickr uses user's locale so the new model should be using the locale: `model = new Date(2022, 1, 25)`